modules/serverless-install-serving-web-console.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+//  * serverless/installing_serverless/installing-knative-serving.adoc
+
+[id="serverless-install-serving-web-console_{context}"]
+= Installing Knative Serving using the web console
+
+.Procedure
+. Navigate in the web console to the *Catalog* → *OperatorHub* → *Installed Operators* page.
```

Comment:
Reword this to keep consistency with a similar step above:
In the {product-title} web console, navigate to the *Catalog* → *OperatorHub* → *Installed Operators* page.
https://github.com/openshift/openshift-docs/pull/21074/files#diff-aa00c6cbca126bf4d6ddf98f09f49e5aR25